eng/pipelines/runtime.yml

```diff
@@ -481,6 +481,18 @@ jobs:
       liveRuntimeBuildConfig: release
       liveLibrariesBuildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
 
+- template: /eng/pipelines/installer/installer-matrix.yml
```

Comment:
This is only adding the legs to CI, if we want the runtime packages to be published, this needs to be added to runtime-official.yml as well.